webpack/scenes/ContentViews/Details/Filters/ContentViewFilters.js

```diff
@@ -111,14 +137,26 @@ const ContentViewFilters = ({ cvId }) => {
             {__('Create filter')}
           </Button>
         </SplitItem>
+        <SplitItem>
+          <Dropdown
+            toggle={<KebabToggle aria-label="bulk_actions" onToggle={toggleBulkAction} />}
+            isOpen={bulkActionOpen}
+            isPlain
+            dropdownItems={[
+              <DropdownItem aria-label="bulk_remove" key="bulk_remove" isDisabled={!bulkActionEnabled} component="button" onClick={bulkRemove}>
+                Remove
```

Comment:
You forgot the best part..translation.. 😄

Reply:
Lol!!! Yes I did.. Thank you for this!